test/pkg/integration/integration.go

```diff
@@ -57,25 +65,33 @@ func NewPodExec(config rest.Config, clientset *kubernetes.Clientset) *PodExec {
 func (p *PodExec) PodCopyFile(src string, dst string, containername string) (*bytes.Buffer, *bytes.Buffer, *bytes.Buffer, error) {
 	var in, out, errOut *bytes.Buffer
 	var ioStreams genericclioptions.IOStreams
-	for {
+	for count := 0; ; count++ {
 		ioStreams, in, out, errOut = genericclioptions.NewTestIOStreams()
 		copyOptions := kubectlcp.NewCopyOptions(ioStreams)
 		copyOptions.Clientset = p.Clientset
 		copyOptions.ClientConfig = p.RestConfig
 		copyOptions.Container = containername
 		err := copyOptions.Run([]string{src, dst})
 		if err != nil {
-			if !errors.Is(err, io.EOF) {
-				return nil, nil, nil, fmt.Errorf("Could not run copy operation: %v", err)
+			if !shouldRetry(count, err) {
+				return nil, nil, nil, fmt.Errorf("could not run copy operation: %v", err)
 			}
 			time.Sleep(10 * time.Second)
+			count = count + 1
 			continue
 		}
 		break
 	}
 	return in, out, errOut, nil
 }
 
+func shouldRetry(count int, err error) bool {
+	if count < connectFailureMaxTries {
+		return err.Error() == errorDialingBackendEOF
+	}
```

Comment:
Should we check if the error is errorDialingBackendEOF, then retry?

Reply:
Yes.

Reply:
I haven't seen you change this. 🤔

Reply:
@jenting 
Sorry, I may misunderstand. Can I ask you to rephrase?

Reply:
Never mind, my thought was the same logic
```go
        if err.Error() == errorDialingBackendEOF && count < connectFailureMaxTries {
		return true
	}
        return false
```